capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -47,11 +50,24 @@ func GetAndSendCapabilityStatement(
 	queueName string) error {
 
 	var err error
+	metadataURL := &url.URL{}
+	*metadataURL = *fhirURL
+	metadataURL.Path = path.Join(metadataURL.Path, "metadata")
+
+	wellKnownURL := &url.URL{}
+	*wellKnownURL = *fhirURL
+	wellKnownURL.Path = path.Join(wellKnownURL.Path, "/.well-known/smart-configuration")
```

Comment:
this will need to be updated based on matt's recent PR. shouldn't be too bad. I recommend creating a helper function for the well-known endpoint similar to what he has for metadata.